guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,247 @@
 [id="host-substatus-overview_{context}"]
 = Host Sub-status Overview
 
-A sub-status monitors only part of a host's capabilities.
+A sub-status monitors only a part of host's capabilities.
 
-Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
-There can be more sub-statuses depending on which plugins you add to your {Project}.
+{Project} contains the following sub-statuses:
 
-The *build* sub-status is relevant for managed hosts.
+Configuration::
+This sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
++
+Possible values:
+// TODO: numbers
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Active | OK | 0
+| Pending | OK | 1
+| No changes | OK | 2
+| No reports | OK / Warning | 3
+| Out of sync | Warning | 4
+| Error | Error | 5
+|===
++
+Additional information about the sub-status values:
++
+* *Active*: During the last run, some resources were applied.
+* *Pending*: During the last run, some resources would be applied but your configuration management integration was configured to run in `noop` mode.
+* *No changes*: During the last run, nothing changed.
+* *No reports*: This can be both a *Warning* or *OK* status.
+When there are no reports but the host uses an associated configuration management proxy or the `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
+Otherwise it maps to *OK*.
+* *Error*: This indicates an error during configuration.
+For example, a run failed to install a package.
+* *Out of sync*: A configuration report was not received within the expected interval, based on the `outofsync_interval` setting.
+Reports are identified by an origin and can have different intervals based upon it.
 
-The *configuration* sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
+Build::
+This sub-status is only relevant for managed hosts, hosts registered through global registration, and when {Project} runs in unattended mode.
++
+Possible values:
+// TODO: numbers
```

Comment:
Here it's easier to understand what's going on:
https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/host_status/build_status.rb#L3-L10